### ISSUE TITLE
Running stryker on folder without csproj, returns incorrect message.

### ISSUE THREAD

**Issue by kwlin** [stryker-mutator/stryker-net#439]
When on a folder without csproj files, the stryker cli returns following message:

```
[09:44:42 INF] Time Elapsed 00:00:02.9486362
Stryker.NET failed to mutate your project. For more information see the logs below:

Expected exactly one .csproj file, found more than one. Please fix your project contents
```

**Comment by richardwerkman** [comment]
What do you think the message should be?

**Comment by kwlin** [comment]
more like this

'''
Expected exactly one .csproj file, found none or more than one. Please fix your project contents
'''

**Comment by richardwerkman** [comment]
Removing the upper line could require too much effort. That line is printed for every exception thrown within stryker. We could refine the messages to look something like this:

```
[09:44:42 INF] Time Elapsed 00:00:02.9486362
Stryker.NET encountered one or more problems. For more information see the logs below:

No csproj file found in the current directory. Are you running Stryker.NET within the right directory? 
```

**Comment by kwlin** [comment]
Maybe I wasn't clear about the incorrect message. The current message is

>[09:44:42 INF] Time Elapsed 00:00:02.9486362
>Stryker.NET failed to mutate your project. For more information see the logs below:
>
>Expected exactly one .csproj file, **found more than one**. Please fix your project contents

While there is no any csproj file found in that directory. A more appropriate message could be:

>[09:44:42 INF] Time Elapsed 00:00:02.9486362
>Stryker.NET encountered one or more problems. For more information see the logs below:
>
>Expected exactly one .csproj file, **found none or more than one**. Please fix your project contents

or

>[09:44:42 INF] Time Elapsed 00:00:02.9486362
>Stryker.NET encountered one or more problems. For more information see the logs below:
>
> **No csproj file found or more than one csproj files** in the current directory. Are you running Stryker.NET within the right directory?

**Comment by Mobrockers** [comment]
We already check for this situation:

https://github.com/stryker-mutator/stryker-net/blob/93ca8a5f37e76885ded1c2f4cb7c12123081cc9d/src/Stryker.Core/Stryker.Core/Initialisation/InputFileResolver.cs#L142-L164

Do you happen to have multiple csproj files in a subdirectory of the directory you were running stryker in? We have recently fixed an issue where we were recursively searching subdirectories for projectfiles instead of only the current directory. Issue mentioned #425 and merged in #426
